src/app/(site)/(root)/_components/ReviewList/reviewList.tsx

```diff
@@ -62,9 +62,6 @@ export const ReviewList = () => {
     <div className="max-w-[1200px]">
       {data?.reviews.map((review: any, index: number) => (
         <div key={review.id} className="flex items-center space-x-2">
-          <span className="font-bold text-gray-500">
-            {(page - 1) * limit + index + 1}.
-          </span>
```

Comment:
You have just removed the span. Are You sure that it is ok to delete?

Reply:
This is unnecessary span